pony-sync.md

```diff
@@ -0,0 +1,35 @@
+# Pony Sync
+
+The Pony Sync is a weekly meeting where Pony contributors discuss topics related to Pony, including recently added or modified issues.
+
+## Running the Pony Sync
+
+The Pony Sync takes place as a [Zoom](https://zoom.us) call. In order to run the Pony Sync you must have Zoom installed, and be logged in as `ponylang.main@gmail.com`. A list of recently modified issues can be generated using the [pony-sync-helper]() application; this list should be pasted into the `sync` stream in Zulip, in a topic whose name is the date of the meeting in US date format (for example May 12, 2020 would be "5/12/2020").
+
+### Prerequisites
+
+In order to run the Pony Sync meeting you will need:
+
+1. A GitHub account
+1. A GitHub personal access token for running the `pony-sync-helper` program
+1. The [`pony-sync-helper`](https://github.com/aturley/pony-sync-helper) program
+1. [Zulip](https://zulipchat.com/) and a Zulip account
+1. A 1Password account that is connected to the Pony 1Password account
+1. [Zoom](https://zoom.us)
+1. The Zoom password for the `ponylang.main@gmail.com` account, which is available in the Pony 1Password vault
+1. Access to the Pony Sync recording account on [nearlyfreespeach.org](https://nearlyfreespeach.org)
+1. A program for transfering files via `scp`
+
+### Steps
+
+1. Generate the list of issues for discussion using the `pony-sync-helper` program. The program repo contains instructions for building and running it.
+1. Using the `ponylang.main@gmail.com` account, log into the Zoom meeting a few minutes before the official start time.
+1. When you feel that there are enough people in the meeting to start, announce that you will be recording the meeting and give participants a few seconds to stop talking. Give a countdown to when you will start recording so that noone is caught unaware.
+1. Start recording the meeting in Zoom.
+1. Begin the meeting by stating that this is the Pony Sync meeting, giving the date, and giving everyone in the room a chance to introduce themselves.
+1. After the introduction the meeting runner can set the agenda and proceed with the meeting.
+1. When the meeting discussion has ended, stop recording and announce that you have stopped recording.
+1. The recording of the meeting will be written to disk when you leave the meeting.
+1. Use SCP to copy the meeting recording to the `r` directory of the Pony account on [nearlyfreespeach.org](https://nearlyfreespeach.org). The file should be called `DATE.m4a`, where `DATE` is the date in YYYY\_MM\_DD format(for example May 12, 2020 would be `2020_05_12.m4a`).
```

Comment:
I think we should add something to this document for how to get an account so you can do the nearlyfreespeech upload.

I think that involves me, do you have access to add others Andy?

Reply:
It doesn't look like I can add others.

Reply:
I guess that means it has to go through me. A note to that effect would be good.

Reply:
Done.